owl_msgs/msg/ArmPose.msg

```diff
@@ -0,0 +1,2 @@
+Header header
```

Comment:
Do we require the NASA license header in these files? Will owl_msgs be under the NASA license?

Reply:
We haven't been putting the header in our other message files. I don't know how important it is, but it should be a separate issue if we decide to do it.

Reply:
NASA license info, no.   We've never put those in message files.  

The ROS Header, good question.  We're not consistent here.  I like the brevity of messages without the header, but I wonder if they are useful enough to add.  Could discuss in Dev Day.